### PR TITLE
Adding Zone 2 dataset

### PR DESCRIPTION
**Pull request by ssadhu-sl**
Adding zone 2 dataset to the collection Included flood-risk-level and flood-risk-type as well.

Reference ticket: https://trello.com/c/keRruoB8/344-adding-flood-risk-zone-data-spike

### CODE REVIEW

File: .github/workflows/run.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Call Collection Run
+on:
+  schedule:
```
Comment (eveleighoj) [inline .github/workflows/run.yml]: maybe remove the cron line so it doesn't run nightly for now?

File: makerules/pipeline.mk [hunk 2]
```diff
@@ -79,11 +79,10 @@ PIPELINE_CONFIG_FILES=\
 	$(PIPELINE_DIR)convert.csv\
 	$(PIPELINE_DIR)default.csv\
 	$(PIPELINE_DIR)default-value.csv\
-	$(PIPELINE_DIR)filter.csv\
 	$(PIPELINE_DIR)lookup.csv\
 	$(PIPELINE_DIR)patch.csv\
 	$(PIPELINE_DIR)skip.csv\
-	$(PIPELINE_DIR)transform.csv
```
Comment (eveleighoj) [inline makerules/pipeline.mk]: this is controlled centrally so don't remove from here, instead add an empty transform.csv

File: pipeline/column.csv [hunk 3]
```diff
@@ -1 +1,2 @@
 dataset,resource,column,field
+flood-risk-zone,,objectid,reference
```
Comment (eveleighoj) [inline pipeline/column.csv:2]: Might not need solving now but we should reach out to the data provider to check if the object ID is a UID that doesn't change between releases

File: collection/endpoint.csv [hunk 4]
```diff
@@ -1 +1,5 @@
 endpoint,endpoint-url,parameters,plugin,entry-date,start-date,end-date
+710664365802fd3664adb099080a4b996834888deb34ebd4c62c40cbd0e45776,https://environment.data.gov.uk/arcgis/rest/services/EA/FloodMapForPlanningRiversAndSeaFloodZone2/MapServer/0,,arcgis,2023-08-23T10:10:49:32Z,,
```
Comment (eveleighoj) [inline collection/endpoint.csv:2]: Did you. attempt to pull out a GeoJSON from arcGIS?

Comment (eveleighoj) [reply collection/endpoint.csv:2]: Again maybe not for now

**Review by eveleighoj — APPROVED** [review]

File: collection/endpoint.csv (hunk 4 above)
Comment (ssadhu-sl) [reply collection/endpoint.csv:2]: The downloadable format in the arcgis link was json which I tried consuming directly but it was not processing. I ll try maybe later again.

**Pull request merged**